roxie/ccd/ccdprotocol.cpp

```diff
@@ -1906,9 +1901,10 @@ class RoxieSocketWorker : public ProtocolQueryWorker
 
                 uid = NULL;
                 sanitizeQuery(queryPT, queryName, sanitizedText, httpHelper, uid, isBlind, isDebug);
-                if (uid)
-                    msgctx->setTransactionId(uid, nullptr, false);
-                else
+
+                msgctx->startSpan(uid, httpHelper.queryRequestHeaders());
```

Comment:
I'm not too clear on how this interacts with or replaces the span created above, currently line 1801:

msgctx->startSpan(nullptr, httpHelper.queryRequestHeaders());

Reply:
Should this really just be setting attributes on the initial span.  For example if necessary updating the global-id header or related attributes?

Reply:
I meant to delete the 1st one - I will fix.  You should not be able to update the global-id/trace-id once the span has been started.  Other properties are ok though.